pkg/bootkube/bootkube.go

```diff
@@ -88,6 +90,10 @@ func makeAPIServerFlags(config Config) []string {
 		"--tls-private-key-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerKey),
 		"--tls-cert-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerCert),
 		"--client-ca-file=" + filepath.Join(config.AssetDir, asset.AssetPathCACert),
+		"--token-auth-file=" + filepath.Join(config.AssetDir, asset.AssetPathBootstrapAuthToken),
+		"--authorization-mode=ABAC,RBAC",
+		"--authorization-policy-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerAuthorizationPolicy),
+		"--authorization-rbac-super-user=admin",
```

Comment:
Let's actually drop the use of this flag. It's being deprecated later. Uses of this should just go in the `system:masters` group.

The bootstrapping policies are here. They're a little hard to read (later releases check in yaml versions), but they summarize the idea.

https://github.com/kubernetes/kubernetes/blob/v1.5.2/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go
https://github.com/kubernetes/kubernetes/blob/v1.5.2/pkg/auth/user/user.go#L69-L78